lib/utils.js

```diff
@@ -209,5 +209,5 @@ module.exports = {
   updateSourceLocation,
   isLicenseFile,
   normalizeSpdx,
-  uniqueOnly
+  filterForUniqueItemsOnly
```

Comment:
@moranthomas Better, even tho I think the extra "Only" in the naming might be not necessary. No worries about that for now.